code/client/munkilib/appleupdates.py

```diff
@@ -640,7 +640,7 @@ def _GetPreferredLocalization(self, list_of_localizations):
         localizations. Returns a single language/localization name.'''
         # pylint: disable=no-self-use
         localization_preferences = (
-            munkicommon.pref('AppleSoftwareUpdateLanguages') or ['English'])
+            munkicommon.getAppleLanguages() or ['English'])
```

Comment:
Not really liking this change. Munki, running as root, needs a definitive set of languages to get. If instead we tie it to the currently logged-in user, that could change: first root's languages, when no one is logged in, then some logged in user's languages; then a different logged-in user's languages...